src/components/ProjectsCard/Project.styled.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  styled,
+  Card,
+  Button,
+  CardContent,
+  Typography,
+} from '@mui/material';
+import DashboardIcon from '@mui/icons-material/Dashboard';

```

Comment:
replace it with the following icon from [react-icons](https://react-icons.github.io/react-icons/search?q=Dashboard)
```suggestion
import { MdDashboard } from 'react-icons/md';
```

Reply:
ok, done.